examples/auth_advanced/advanced_client.go

```diff
@@ -89,7 +95,7 @@ type MyTokenCache struct {
 	mu          sync.Mutex
 }
 
-func (s *MyTokenCache) RetrieveToken(ctx context.Context) (*string, error) {
+func (s *MyTokenCache) RetrieveToken(_ context.Context) (*string, error) {
```

Comment:
why return *string instead of string?

Reply:
Good comment. I will move interface from `*string` to `string`. This will mean we will use `""` as unset token instead of nil.

Reply:
Changed